docs/dev_guide/index.rst

```diff
@@ -20,3 +20,4 @@ We currently recommend the :ref:`newcomers` as the place to start.
    config
    documentation
    tests
+   stability
```

Comment:
I am not sure the dev guide is the right place for this either.... :confused:

Reply:
Problem is, I  dont know where to put it.

Reply:
me either, let's leave it here for now and give it some thought.